src/java.base/share/classes/java/net/NetworkInterface.java

```diff
@@ -547,27 +507,15 @@ public boolean supportsMulticast() throws SocketException {
 
     /**
      * Returns the hardware address (usually MAC) of the interface if it
-     * has one and if it can be accessed given the current privileges.
+     * has one.
      *
      * @return  a byte array containing the address, or {@code null} if
-     *          the address doesn't exist or is not accessible
+     *          the address doesn't exist
```

Comment:
"accessed given the current privileges" relates to OS privileges/capabilities so not security manager related. So I think drop the change the method description, only change to this method should be the removal of the SM permission check.

Reply:
Okay, so "not only security manager related then" :-) The current implementation does return null on checkPermission failure, but yes, there are other ways null may be returned.

I'll remove the method description changes and update the CSR as well.

Reply:
> Okay, so "not only security manager related then" :-) The current implementation does return null on checkPermission failure, but yes, there are other ways null may be returned.

JEP 486 removed the sentence "If a security manager is set .." and changed the return description. There are platforms where the process needs privileges or capabilities to read the hardware address so that the motive for the description. Thanks for dropping this change from the PR/CSR.